test/unit/data/schema/integrity_spec.js

```diff
@@ -3,31 +3,36 @@ const _ = require('lodash');
 const crypto = require('crypto');
 const schema = require('../../../../core/server/data/schema');
 const fixtures = require('../../../../core/server/data/schema/fixtures');
+const defaultSettings = require('../../../../core/server/data/schema/default-settings');
 
 /**
  * @NOTE
  *
- * If this test fails for you, you have modified the database schema or fixtures.
+ * If this test fails for you, you have modified the database schema or fixtures or default settings.
  * When you make a change, please test that:
  *
  * 1. A new blog get's installed and the database looks correct and complete.
  * 2. A blog get's updated from a lower Ghost version and the database looks correct and complete.
+ * 3.
```

Comment:
but what is 3? :grin:

Reply:
Haha, realized half-way its not needed :P Updated